rust-toolchain

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.64.0"
+channel = "stable"
```

Comment:
aligns with ref-fvm and allows compilation of a dependency that does not build on 1.64.0